### PR TITLE
Migrate `azure.timeseries` to swan lake preview 1

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

### CODE REVIEW

File: src/azure.timeseries/constants.bal [hunk 1]
```diff
@@ -14,10 +14,16 @@
...
 const string AZURE_LOGIN_BASE_URL = "https://login.microsoftonline.com/";
 const string AZURE_TSI_DEFAULT_SCOPE = "https://api.timeseries.azure.com//.default";
+
+const string ENVIRONMENTS_API = "Get Environments API";
+const string AVAILABILITY_API = "Get Environment Availability API";
+const string METADATA_API = "GGet Environment Metadata API";
+const string EVENTS_API = "Get Environment Events API";
+const string AGGREGATES_API = "Get Environment Aggregates API";
+
+type TIMESERIES_API ENVIRONMENTS_API|AVAILABILITY_API|METADATA_API|EVENTS_API|AGGREGATES_API;
```
Comment (kalaiyarasiganeshalingam) [inline src/azure.timeseries/constants.bal:29]:
```suggestion
type TIMESERIES_API ENVIRONMENTS_API|AVAILABILITY_API|METADATA_API|EVENTS_API|AGGREGATES_API;

```

File: src/azure.timeseries/utils.bal [hunk 2]
```diff
@@ -12,15 +12,20 @@
 // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
+import ballerina/http;
 
-function processResError(error errorResponse) returns @untainted error {
-    error err = error(ERROR_CODE, message = <string>errorResponse.detail()?.message);
-    return err;
+function processErrorResponse(error errorResponse, TIMESERIES_API apiName) returns @untainted error {
+    return error("Error invoking '" + apiName.toString() + "'.", cause = errorResponse);
```
Comment (chamil321) [inline src/azure.timeseries/utils.bal:18]:
```suggestion
    return error("Error invoking '" + apiName.toString() + "'.", errorResponse);
```

**Comment by daneshk** [comment]
@niveathika Shall we change the version to `2.0.0` please. for the swan lake releases

**Comment by niveathika** [comment]
> @niveathika Shall we change the version to `2.0.0` please. for the swan lake releases

@daneshk Bumped in 8e68a3e

**Comment by niveathika** [comment]
Closing & reopening for secrets update

**Review by chamil321 — APPROVED** [review]

**Pull request merged**